src/options.ts

```diff
@@ -123,8 +139,10 @@ export class Options {
       items: 'zodiac-item',
       track: 'zodiac-track',
     },
+    enableLiveRegion: true,
     gap: 8,
     itemsPerView: 5,
+    liveRegionText: 'Slide @position of @total @title',
```

Comment:
Should we have some punctuation between the positional information and the active item title? Perhaps a colon?

Reply:
Defer to Steven's judgement. Marked as resolved since this isn't a major blocker.